docs/Config_Changes.md

```diff
@@ -6,6 +6,9 @@ All dates in this document are approximate.
 
 # Changes
 
+20210501: The `resonance_tester` module no longer requires modifying
+`max_accel` parameter for resonance testing.
+
```

Comment:
I'm not sure we need to document this change, because the change is backwards compatible.  (That is, users could still use the old method.)  I guess if you wanted to we could document that the TEST_RESONANCES command will now increase max_accel beyond the configured limit.  I don't think it's needed though.

Reply:
OK, I've removed this notice.